cmd/estuary-shuttle/main.go

```diff
@@ -1149,12 +1149,8 @@ func (s *Shuttle) handleLogLevel(c echo.Context) error {
 func (s *Shuttle) handleAdd(c echo.Context, u *User) error {
 	ctx := c.Request().Context()
 
-	if u.StorageDisabled || s.disableLocalAdding {
-		return &util.HttpError{
-			Code:    http.StatusBadRequest,
-			Reason:  util.ERR_CONTENT_ADDING_DISABLED,
-			Details: "uploading content to this node is not allowed at the moment",
-		}
+	if err := util.ErrorIfContentAddingDisabled(u.StorageDisabled || s.disableLocalAdding); err != nil {
```

Comment:
```suggestion
	if err := util.ErrorIfContentAddingDisabled(s.isContentAddingDisabled(u)); err != nil {
```

Reply:
shuttle cant use `isContentAddingDisabled`

Reply:
We could just add the function:

```
func (s *Shuttle) isContentAddingDisabled(u *User) bool {
	return s.disableLocalAdding || u.StorageDisabled
}
```

Reply:
thank you